### PR TITLE
Centos 6 and RHEL 6 to use virtio bus for network and storage by default

### PR DESCRIPTION
**Pull request by enp0s3**
Old guests can now use virtio bus for disks and network interfaces
therefore configure the template to use that bus type as a default bus for them.

Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**Release note**:
```release-note
Centos 6 and RHEL 6 based images to consume virtio bus for disks and network devices
```

### CODE REVIEW

File: templates/centos6.tpl.yaml [hunk 1]
```diff
@@ -119,7 +119,7 @@ objects:
             interfaces:
             - masquerade: {}
               name: default
-              model: e1000e
+              model: virtio
```
Comment (ksimon1) [inline templates/centos6.tpl.yaml:122]: Shouldn't be this change in disk section? Because now you are updating network interface

Comment (enp0s3) [reply templates/centos6.tpl.yaml:122]: @ksimon1 I've updated the PR subject and description. The aim of this PR is to swtich both disk and network devices to work with virtio.

**Comment by enp0s3** [comment]
/hold
Need to validate with latest centos 6 image on plain kubevirt

**Comment by dankenigsberg** [comment]
As this changes user-facing behavior, a release note is needed.

Given virtio is back, I suggest to reintroduce  `networkInterfaceMultiqueue: true`

**Comment by enp0s3** [comment]
> As this changes user-facing behavior, a release note is needed.
> 
> Given virtio is back, I suggest to reintroduce `networkInterfaceMultiqueue: true`

@dankenigsberg Done

**Comment by enp0s3** [comment]
/retest
the fix was merged to KV release 0.37

**Comment by enp0s3** [comment]
/retest
```
export KUBEVIRT_VERSION=$(_curl -L https://api.github.com/repos/kubevirt/kubevirt/releases | \
            jq '.[] | select(.prerelease==false) | .name' | sort -V | tail -n1 | tr -d '"')
```
Release tag ready and I've verified that now ```KUBEVIRT_VERSION``` is  ```v0.37.2```

**Pull request merged**